lonabot/utils.py

```diff
@@ -43,25 +43,29 @@ class NoDeltaError(ValueError):
 _DUE_TIME = re.compile(fr'{_F}:{_F}(?::{_F})?')
 
 
-def parse_when(when, time_delta: TimeDelta, utc_now):
+def parse_when(when, time_delta: TimeDelta, utc_now, later_delta=None):
     # Returns `due` for either `delay` or `due`.
     due, text = parse_due(when, time_delta, utc_now)
     if due:
         return due, text
 
-    delay, text = parse_delay(when)
+    delay, text = parse_delay(when, later_delta)
     if delay:
         due = int(utc_now.timestamp()) + delay
         return due, text
 
     return None, None
 
 
-def parse_delay(when):
+def parse_delay(when, later_delta=None):
     iso = _parse_delay_iso(when)
     if iso:
         return iso
 
+    later = when.split(maxsplit=1)
+    if later[0].upper() == 'LATER' and later_delta is not None:
```

Comment:
We're not angry, our users are:

```suggestion
    if later[0].lower() == 'later' and later_delta is not None:
```

Reply:
I used upper because coming from C#, doing upper is recommended due to unicode reasons or something. Is that not the case in Python?

Reply:
In Python we have [`str.casefold`](https://docs.python.org/3/library/stdtypes.html#str.casefold) which converts to lowercase. But we're comparing against ASCII characters anyway so I don't think unicode matters much here.

Reply:
It now uses `str.casefold()` to be in line with what Python docs recommend.